compiler/src/main/java/bullet/impl/ComponentProcessingStep.java

```diff
@@ -145,6 +150,15 @@ private String getName(TypeMirror type) {
 
     final ClassName elementName = ClassName.get(element);
 
+    // Filter out duplicate injections
```

Comment:
The issue it that you may have both:

    void inject(Foo f);
    MembersInjector<Foo> getFooInjector();

which is not "doing it wrong" if you have both, you pick the one that is more optimal, which is the members injector.